src/Components/RecommendeeCard/_RecommendeeCard.scss

```diff
@@ -1,7 +1,25 @@
 .recommendee-card-container{
-    width: 800px;
+    width: 900px;
```

Comment:
I can walk you through changing this into sass if you'd like. We can do it now or after MVP. Just let me know! 😃

Reply:
Yeah lets do scss sometime!

Reply:
Sounds Great!